### ISSUE TITLE
[Discussion] Usage of JS editor for Web API examples

### ISSUE THREAD

**Issue by NiedziolkaMichal** [mdn/interactive-examples#2349]
I would like to propose the usage of a JavaScript editor, for Web API functions that don't require any usage of HTML or CSS. It would be useful for example in [Encoding](https://developer.mozilla.org/en-US/docs/Web/API/Encoding_API), [Performance](https://developer.mozilla.org/en-US/docs/Web/API/Performance_API), [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL), many properties of [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window), [Navigator](https://developer.mozilla.org/en-US/docs/Web/API/Navigator), [Screen](https://developer.mozilla.org/en-US/docs/Web/API/Screen), and more.
Here is an example of how it could look for Screen.height:
![image](https://user-images.githubusercontent.com/100634371/205454954-022e3790-0b90-46bf-a3af-8b7339f1e189.png)

@schalkneethling @wbamberg @Rumyra @bsmth @teoli2003 @nschonni @estelle Do you agree?

**Comment by bsmth** [comment]
Makes sense to me, I'd be curious what the rest of the team thinks, too!

**Comment by wbamberg** [comment]
I think this is a good idea! It would be good to do some analysis of which pages/APIs would benefit. I don't think the specific example you have here is a very good case (you can't actually change it to anything else and see a different result, which rather defeats the point of it being interactive). I think `Performance` would be tricky as well.

But for instance I can see it being useful for interfaces like `URL` or `URLPattern`.
